sunspot/lib/sunspot/schema.rb

```diff
@@ -21,9 +21,6 @@ class Schema #:nodoc:all
       FieldType.new('string', 'Str', 's'),
       FieldType.new('sdouble', 'SortableDouble', 'e'),
       FieldType.new('slong', 'SortableLong', 'l'),
```

Comment:
Should the point type be added here now?

Reply:
I'm not sure what this file actually is for - the Sortable* types are deprecated and even removed since Solr 5...

Reply:
I know this might be a bold suggestion, but in times of dockerization, virtual servers and clouds we might think about a) dropping updating the sunspot_solr gem altogether and b) just supplying predefined Solr configs that work with sunspot. At least that's the way we use it since years now. But that's really only meant as a suggestion based on personal usage.